src/OSPSuite.Presentation/Presenters/Importer/ImporterDataPresenter.cs

```diff
@@ -49,11 +48,9 @@ public void ImportDataForConfirmation()
          var sheets = new Cache<string, DataSheet>();
          foreach (var element in _dataSourceFile.DataSheets.KeyValues)
          {
-            if (!Sheets.Keys.Contains(element.Key))
-            {
-               Sheets.Add(element.Key, element.Value);
-               sheets.Add(element.Key, element.Value);
-            }
+            if (Sheets.Keys.Contains(element.Key)) continue;
+            Sheets.Add(element.Key, element.Value);
+            sheets.Add(element.Key, element.Value);
          }
 
          if (sheets.Count == 0) return;
```

Comment:
same with return when they are in the middle of functions

Reply:
I fixed a couple already. I think it is good to go now. Thanks for the catch.